spring-vault-core/src/main/java/org/springframework/vault/core/lease/domain/Lease.java

```diff
@@ -42,14 +44,14 @@ private Lease(String leaseId, long leaseDuration, boolean renewable) {
 	/**
 	 * Create a new {@link Lease}.
 	 *
-	 * @param leaseId must not be empty or {@literal null}.
+	 * @param leaseId must not {@literal null}. Empty string implies a generic secret.
```

Comment:
the parameters are all of different types, so I don't see the harm in reusing `of`. will push this change up shortly